packages/devtools_app/integration_test/run_tests.dart

```diff
@@ -52,15 +51,9 @@ void main(List<String> args) async {
         await _runTest(newArgsWithTarget);
       }
     }
-  } on Exception catch (e) {
-    exception = e;
```

Comment:
please don't remove all this exception handling.

Reply:
We discussed that this exception handling does not change behavior.